File: Assignment5/Program4.c
// 4.Write a program which accept number from user and count frequency of 4 in it.
// Input :  2395  Output :  0
// Input :  1018 Output :  0
// Input :  9440 Output :  2
// Input :  922432 Output :  1
#include <stdio.h>
int CountFour(int);
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter number");
    scanf("%d", &iValue);

    iRet = CountFour(iValue);
    printf("%d ", iRet);
    return 0;
}

int CountFour(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    while (iNo > 0)
    {
        iDigit = iNo % 10;
        iNo = iNo / 10;
        if (iDigit == 4)
        {
            iCnt++;
        }
    }
    return iCnt++;
}

File: Assignment33/Program4.java
// 4.Write a program which accept number from user and return multiplication of all digits. 
// Input :  2395  Output :  270 
// Input :  1018 Output :  8 
// Input :  9440 Output :  144 
// Input :  922432 Output :  864 

import java.util.*;


class Program4
{
	public static void main(String[] args) 
	{
		Scanner sobj = new Scanner(System.in);
		int iValue = 0,iRet = 0;

		System.out.println("Enter number");
		iValue = sobj.nextInt();

		Digit dobj = new Digit();
		iRet = dobj.Multiply(iValue);

		System.out.println("Multipication of digits are :"+iRet);
	}
}

class Digit 
{  
	public int Multiply(int iNo)  
	{
		int iDigit = 0,iMul = 1;

		while(iNo != 0)
		{
			iDigit = iNo % 10;
			if(iDigit == 0)
			{
				iDigit++;
			}
			iMul = iMul * iDigit;
			iNo = iNo / 10;
		}
		return iMul;
	} 
}